cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -336,7 +336,10 @@ func getControllerManagerCommand(cfg *kubeadmapi.MasterConfiguration, k8sVersion
 	// Let the controller-manager allocate Node CIDRs for the Pod network.
 	// Each node will get a subspace of the address CIDR provided with --pod-network-cidr.
 	if cfg.Networking.PodSubnet != "" {
-		maskSize := calcNodeCidrSize(cfg.Networking.PodSubnet)
+		maskSize := cfg.Networking.NodeCIDRMaskSize
+		if maskSize == "" {
```

Comment:
I'm wondering if setting default should go here https://github.com/jstangroome/kubernetes/blob/438818f22932c08b86ebaa7de5c7c5db9d6d7d7c/cmd/kubeadm/app/util/config/masterconfig.go#L38

since that's where we do this sort of thing